db/migrate/20240315160350_rename_adopter_accounts_to_adopter_foster_accounts.rb

```diff
@@ -0,0 +1,7 @@
+class RenameAdopterAccountsToAdopterFosterAccounts < ActiveRecord::Migration[7.0]
+  def change
+    if table_exists?(:adopter_account)
+      rename_table :adopter_account, :adopter_account
```

Comment:
table names should be plural I believe and the second `adopter_account` needs to be `adopter_foster_accounts`